app/utils/client/lib/roomTypes.js

```diff
@@ -67,6 +69,22 @@ export const roomTypes = new class RocketChatRoomTypes extends RoomTypesCommon {
 		return ChatSubscription.find({ rid }).count() > 0;
 	}
 
+	emitNewMessageNotification(rid) {
+		const roomType = this.getRoomType(rid);
+		const newMessageNotificationFunction = this.roomTypes[roomType || 'l'].emitNewMessageNotification;
```

Comment:
```suggestion
		const newMessageNotificationFunction = this.roomTypes[roomType || 'l'].emitNewMessageNotification;
```

We can't hard code `l`,  just the `roomType` should be enough.